src/gui/transferlistwidget.cpp

```diff
@@ -1179,7 +1179,8 @@ void TransferListWidget::currentChanged(const QModelIndex &current, const QModel
     {
         torrent = m_listModel->torrentHandle(mapToSource(current));
         // Scroll Fix
-        scrollTo(current);
+        // Delay the call to scrollTo until control is passed back to the event queue
+        QMetaObject::invokeMethod(this, [this, current] { scrollTo(current); }, Qt::QueuedConnection);
```

Comment:
There is no need to write comments that simply "translate" from C++ to English. It is better to explain why this is necessary so that no one in the future will have a desire to "simplify" this code.

Reply:
replaced the "why" comment